### PR TITLE
LU-7710-duplicate-ids

### PR DESCRIPTION
**Pull request by pds2208**

### CODE REVIEW

File: blaise_survey_manager/survey_config/templates/list_surveys.html [hunk 1]
```diff
@@ -44,9 +44,9 @@ <h1>Surveys</h1>
         {% else %}
             {% for survey in surveys %}
                 <tr class="table__row">
-                    <td class="table__cell" id="survey-{{ survey.Survey.tla }}">{{ survey.Survey.tla }}</td>
-                    <td class="table__cell" id="survey-{{ survey.Survey.survey_name }}">{{ survey.Survey.survey_name }}</td>
-                    <td class="table__cell" id="instrument-count">{{ survey.total_instruments }}
+                    <td class="table__cell" id="survey-{{ survey.Survey.id }}">{{ survey.Survey.tla }}</td>
+                    <td class="table__cell" id="survey-name-{{ survey.Survey.id }}">{{ survey.Survey.survey_name }}</td>
+                    <td class="table__cell" id="instrument-count={{  survey.Survey.id  }}">{{ survey.Survey.id }}
```
Comment (elthorne) [inline blaise_survey_manager/survey_config/templates/list_surveys.html:49]: this should be the count of instruments not the Survey ID.

In this example https://dev-el-37-bsm.social-surveys.gcp.onsdigital.uk/survey_config/list_surveys LFS has 0 instruments yet OPN has 2.

Comment (pds2208) [reply blaise_survey_manager/survey_config/templates/list_surveys.html:49]: Doh! Fixed

Comment (elthorne) [reply blaise_survey_manager/survey_config/templates/list_surveys.html]: <3

**Review by elthorne — APPROVED** [review]
Tested in sandbox and seems all ok.

**Pull request merged**